proto/ibc/applications/fee/v1/tx.proto

```diff
@@ -48,6 +55,24 @@ message MsgRegisterCounterpartyAddress {
 // MsgRegisterCounterpartyAddressResponse defines the response type for the RegisterCounterpartyAddress rpc
 message MsgRegisterCounterpartyAddressResponse {}
 
+// MsgRegisterDistributionAddress defines the request type for the RegisterDistributionAddress rpc
+message MsgRegisterDistributionAddress {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  // the relayer address
+  string address = 1;
+  // the fee distribution address
+  string distribution_address = 2 [(gogoproto.moretags) = "yaml:\"distribution_address\""];
+  // unique port identifier
+  string port_id = 3 [(gogoproto.moretags) = "yaml:\"port_id\""];
```

Comment:
Also a nit: should we change the order of the parameters to `port_id`, `channel_id`, `address`, `distribution_address`? Also for the register counterparty address command. I think in other messages the port and channel are the first parameters...

Reply:
Done!